src/sentry/services/rpc/protobufs/Organization.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+import "sentry/services/rpc/protobufs/User.proto";
+
+service OrganizationService {
+    rpc ChangeName(ChangeNameRequest) returns (Organization) {}
+}
+
+message Organization {
+    int32 id = 1;
+    string name = 2;
```

Comment:
We would need to figure out where authentication/identity context goes into these request payloads.